graphistry/Plottable.py

```diff
@@ -1,18 +1,22 @@
-from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Set, Tuple, Union
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Set, Tuple, Union, Protocol
 from typing_extensions import Literal
 import pandas as pd
 
 from graphistry.models.ModelDict import ModelDict
 from graphistry.models.compute.chain_remote import FormatType, OutputTypeAll, OutputTypeDf, OutputTypeGraph
 from graphistry.models.compute.dbscan import DBSCANEngine
 from graphistry.models.compute.umap import UMAPEngineConcrete
+from graphistry.models.compute.features import GraphEntityKind
 from graphistry.plugins_types.cugraph_types import CuGraphKind
+from graphistry.plugins_types.graphviz_types import EdgeAttr, Format, GraphAttr, NodeAttr, Prog
 from graphistry.plugins_types.kusto_types import KustoConfig
 from graphistry.plugins_types.spanner_types import SpannerConfig
+from graphistry.plugins.kustograph import KustoGraph
+from graphistry.plugins.spannergraph import SpannerGraph
+from graphistry.privacy import Mode as PrivacyMode
```

Comment:
hmm how do we avoid circular imports at the TYPE_CHECKING and dynamic phases now?

Reply:
yes, fixed with runtimecheckable